src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import FormButtons from '../components/FormButtons';
+import { getFormPageInfo } from '../utils/selectors';
+import { PURPOSE_TEXT } from '../utils/constants';
+
+const initialSchema = {
+  type: 'object',
+  required: ['reasonForAppointment'],
+  properties: {
+    reasonForAppointment: {
+      type: 'string',
+      enum: [
+        PURPOSE_TEXT['routine-follow-up'],
+        PURPOSE_TEXT['new-issue'],
+        PURPOSE_TEXT['medication-concern'],
+      ],
+    },
+  },
+};
+
+const uiSchema = {
+  reasonForAppointment: {
+    'ui:title': 'Why do you want to make an appointment?',
```

Comment:
This line can be removed since we set `hideLabelText: true` below.

Reply:
Done